easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.13-foss-2021a-R-4.1.0.eb

```diff
@@ -1053,6 +1053,15 @@ exts_list = [
     ('JASPAR2020', '0.99.10', {
         'checksums': ['b9b92d141a317ebb32a14708229f6b82522ceeb5f1b88360d93b0a7cfe30375b'],
     }),
+    ('AUCell', '1.14.0', {
+        'checksums': ['5936a7b33d7afb144e4bcfb99bece7de51d01063804c9032107132aaf060aa61'],
+    }),
+    ('arrow', '6.0.0.2', {
```

Comment:
[`arrow` install instructions](https://arrow.apache.org/docs/r/articles/install.html). It looks like we should be able to use an existing eb install of Arrow here.

Reply:
[My failure was](https://github.com/easybuilders/easybuild-easyconfigs/pull/14259#issuecomment-955673490):
```
* installing *source* package arrow ...
** package arrow successfully unpacked and MD5 sums checked
** using staged installation
*** Found local C++ source: 'tools/cpp'
*** Building C++ libraries
Error in if (nzchar(path) && cmake_version(path) >= version_required) { : 
  missing value where TRUE/FALSE needed
Calls: build_libarrow -> ensure_cmake -> find_cmake
In addition: Warning message:
In system(paste(cmd, "--version"), intern = TRUE, ignore.stderr = TRUE) :
  running command '/usr/bin/cmake --version 2>/dev/null' had status 1
Execution halted
------------------------- NOTE ---------------------------
There was an issue preparing the Arrow C++ libraries.
See https://arrow.apache.org/docs/r/articles/install.html
---------------------------------------------------------
ERROR: configuration failed for package arrow
```

Reply:
fixed in 32c4221